content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -165,12 +167,135 @@ This translates to the following SQL commands:
+| `NumericMinMaxOps`          | `Decimal` (`@db.Decimal`)            | ✅      |                               |
+| `NumericMinMaxMultiOps`     | `Decimal` (`@db.Decimal`)            |         | 14+                           |
+| `OidBloomOps`               | `Int` (`@db.Oid`)                    |         | 14+                           |
+| `OidMinMaxOps`              | `Int` (`@db.Oid`)                    | ✅      |                               |
+| `OidMinMaxMultiOps`         | `Int` (`@db.Oid`)                    |         | 14+                           |
+| `TextBloomOps`              | `String` (`@db.Text`, `@db.VarChar`) |         | 14+                           |
+| `TextMinMaxOps`             | `String` (`@db.Text`, `@db.VarChar`) | ✅      |                               |
+| `TextMinMaxMultiOps`        | `String` (`@db.Text`, `@db.VarChar`) |         | 14+                           |
+| `TimestampBloomOps`         | `DateTime` (`@db.Timestamp`)         |         | 14+                           |
+| `TimestampMinMaxOps`        | `DateTime` (`@db.Timestamp`)         | ✅      |                               |
+| `TimestampMinMaxMultiOps`   | `DateTime` (`@db.Timestamp`)         |         | 14+                           |
+| `TimestampTzBloomOps`       | `DateTime` (`@db.Timestamptz`)       |         | 14+                           |
+| `TimestampTzMinMaxOps`      | `DateTime` (`@db.Timestamptz`)       | ✅      |                               |
+| `TimestampTzMinMaxMultiOps` | `DateTime` (`@db.Timestamptz`)       |         | 14+                           |
+| `TimeBloomOps`              | `DateTime` (`@db.Time`)              |         | 14+                           |
+| `TimeMinMaxOps`             | `DateTime` (`@db.Time`)              | ✅      |                               |
+| `TimeMinMaxMultiOps`        | `DateTime` (`@db.Time`)              |         | 14+                           |
+| `TimeTzBloomOps`            | `DateTime` (`@db.Timetz`)            |         | 14+                           |
+| `TimeTzMinMaxOps`           | `DateTime` (`@db.Timetz`)            | ✅      |                               |
+| `TimeTzMinMaxMultiOps`      | `DateTime` (`@db.Timetz`)            |         | 14+                           |
+| `UuidBloomOps`              | `String` (`@db.Uuid`)                |         | 14+                           |
+| `UuidMinMaxOps`             | `String` (`@db.Uuid`)                | ✅      |                               |
+| `UuidMinMaxMultiOps`        | `String` (`@db.Uuid`)                |         | 14+                           |
+| `raw("other")`              |                                      |         |                               |
+
+The operator classes marked to work with CockroachDB are the only ones allowed on that database. On CockroachDB the operator class must be emitted from the field definition.
```

Comment:
> On CockroachDB the operator class must be emitted from the field definition.

What does this mean?

Reply:
You cannot set `ops` in the index field definition. Under the surface it uses the operator classes from the table, but you cannot explicitly tell them in the DDL and the information schema does not return them in the query.

Reply:
```suggestion
The operator classes marked to work with CockroachDB are the only ones allowed on that database. The operator class can further not be changed from the one possible value, `ops` is not necessary or allowed for CockroachDB.
```